docs/appendices/spin.md

```diff
@@ -17,3 +17,5 @@ The Spin-2 system developed by QuTech supports the following subset of the cQASM
 * All other libqasm 1.x language structures are not supported.
 * No `prep_X` and `measure_X` instructions are allowed, the lls will init all supported qubits before circuit execution
   and measure all in the z-basis at the end of the circuit and only return the subset as defined in the qubit register.
+* Bit register declaration statements, _e.g._, `bit[<number-of-bits:INT>] <bit-register-name:ID>` or `bit <bit-name:ID>`, are accepted and subsequently ignored.
+* Measure instruction statements assign outcomes to bit variables, _e.g._ `<bit-name:BIT> = measure <qubit-argument:QUBIT>`. These statements are accepted and subsequently ignored.
```

Comment:
Just for confirmation. We do not have to accept prep_X statements right?

Reply:
Also, `prep_X` is not part of the language specification atm. However, we may introduce the `reset` instruction at some point in the future (which is the [OpenQASM equivalent of the `prep_X`](https://openqasm.com/language/insts.html#initialization)), as other backend require it to describe certain algorithms.